// This proto definition is copied from flower's grpcadapter.proto, and must be kept in sync with it.
syntax = "proto3";

// Change flwr.proto to flower.proto to avoid namespace conflict when both Flower and Flare are installed
package flower.proto;

service GrpcAdapter {
  rpc SendReceive(MessageContainer) returns (MessageContainer) {}
}

message MessageContainer {
  map<string, string> metadata = 1;
  string grpc_message_name = 2;
  bytes grpc_message_content = 3;
}
